### PR TITLE
feat: Update `bazecor` from 1.3.9 to 1.3.10

### PR DESCRIPTION
**Pull request by gabyx**
## Description of changes
- Update `bazecor` to 1.3.10

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [x] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by amesgen — CHANGES_REQUESTED** [review]
Thanks, looks good, only the commit message should be changed to conform to the [commit conventions](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#commit-conventions):

> basecor: 1.3.9 -> 1.3.10

**Comment by gabyx** [comment]
Arent Nixpkgs doing squash merges? So the whole commit convention thing would be moved on the the shoulders of the maintainers. (dont get me wrong I like that)...

**Comment by amesgen** [comment]
Thanks! BTW: bazecor 1.3.11 has been released in the meantime: https://github.com/Dygmalab/Bazecor/releases/tag/v1.3.11

**Comment by gabyx** [comment]
Sure I bumbed it again.

**Comment by gabyx** [comment]
@amesgen Can this be merged? Thanks. ;-)

File: pkgs/applications/misc/bazecor/default.nix
Comment (amesgen) [inline pkgs/applications/misc/bazecor/default.nix:1]: Minor: There are some unnecessary formatting changes, can you revert them?

**Review by amesgen — APPROVED** [review]
> Can this be merged? Thanks. ;-)

I am not a nixpkgs committer, so I can't merge this, but you can post this e.g. here: https://discourse.nixos.org/t/prs-already-reviewed/2617/955

**Comment by gabyx** [comment]
@amesgen: should be good now.

**Comment by NickCao** [comment]
Superseded by https://github.com/NixOS/nixpkgs/pull/304887

**Pull request closed without merging**
